Source/CombatExtended/Compatibility/EDShields.cs

```diff
@@ -102,15 +78,17 @@ public static bool CheckForCollisionCallback(ProjectileCE projectile, IntVec3 ce
                 if ((Quaternion.Angle(targetAngle, shieldProjAng) > 90))
```

Comment:
I don't *think* this is needed anymore (in fact, I think it's detrimental), but wouldn't mind someone who is decent at maths double checking me on this.  If we started outside the circle, and are now inside it, we *must* have been moving toward the center of the circle, which is what this checks for.  If we are sufficiently fast to have shot past the center of the circle, then this check will think we're headed out of the circle (hence it's detrimental).  If we started inside the circle (what this was originally designed to detect), then line 71 will have `continue`d.  Is there any edge case (probably involving standing on the edges or targeting the edges) I missed?